src/frontend/js/data/CourseProductProvider/index.spec.tsx

```
@@ -24,6 +24,7 @@ describe('useCourseProduct', () => {
   });
 
   it('throws an error if it is not used within a CourseProductProvider', () => {
+    jest.spyOn(console, 'error').mockImplementation(noop);
```

Are you sure that you do not need to unmock this at the bottom of this test ? ( are there any risk that it hides error of other following test suites ? )

---

This the purpose of the `jest.restoreAllMocks` within the `afterEach`

---

Right!